site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/FileTable.js

```diff
@@ -0,0 +1,95 @@
+import _get from "lodash/get";
+import PropTypes from "prop-types";
+import React from "react";
+import {
+    Icon,
+    Popup,
+    Table,
+} from "semantic-ui-react";
+import { humanReadableBytes } from "react-invenio-deposit";
+
+
+const FileTableRow = ({ handleDelete, file, decimalSizeDisplay }) => {
+    return (
+        <Table.Row key={file.name}>
+            <Table.Cell data-label={"Filename"} width={10}>
+                <div>
+                    <label>{file.name}</label>
+                    <br />
+                    {file.checksum && (
+                        <div className="ui text-muted">
+                            <span style={{ fontSize: "10px" }}>{file.checksum}</span>{" "}
```

Comment:
inline styling is violating CSP, could you move it to CSS intead?

Reply:
I got this snippet from `react-invenio-deposit` and, in fact, `file.checksum` never exists here. 

Therefore I am removing the checksum block